static/scss/email-composition-dialog.scss

```diff
@@ -1,67 +1,105 @@
-    flex-direction: column;
+.email-composition-dialog-wrapper {
+  overflow: auto;
 
-    .subheading-section {
-      padding-bottom: 20px;
+  .email-composition-dialog {
+    padding-top: 0 !important;
 
-      .subheading-to {
-        padding: 0 10px;
+    > div:first-child > div {
+      height: auto !important;
+      max-height: inherit !important;
+    }
 
-      }
+    .email-composition-contents {
+      display: flex;
+      flex-direction: column;
+      max-height: inherit;
 
-      .subheading {
-        letter-spacing: normal;
-        margin: 0;
-        padding: 0;
-        font-size: 16px;
-        font-weight: normal;
+      .subheading-section {
+        padding-bottom: 20px;
 
-        &.default {
-          font-style: italic;
+        .subheading-to {
+          padding: 0 10px;
         }
 
-        &.rounded {
-          background-color: $bg-med-gray;
-          line-height: 22px;
-          padding: 10px 10px 10px 20px;
-          border-radius: 20px;
-          font-style: normal;
-        }
+        .subheading {
+          letter-spacing: normal;
+          margin: 0;
+          padding: 0;
+          font-size: 16px;
+          font-weight: normal;
+
+          &.default {
+            font-style: italic;
+          }
 
-        &.profile-image-bubble {
-          background-color: $bg-med-gray;
-          border-radius: 50px;
-          padding: 0 30px 0 0;
-          height: 43px;
-          display: inline-block;
+          &.rounded {
+            background-color: $bg-med-gray;
+            line-height: 22px;
+            padding: 10px 10px 10px 20px;
+            border-radius: 20px;
+            font-style: normal;
+          }
 
-          >span {
+          &.profile-image-bubble {
+            background-color: $bg-med-gray;
+            border-radius: 50px;
+            padding: 0 30px 0 0;
+            height: 43px;
             display: inline-block;
-            padding-left: 15px;
+
+            > span {
+              display: inline-block;
+              padding-left: 15px;
+            }
           }
         }
       }
-    }
 
-    .validation-error {
-      font-size: 12px;
-      color: $validation-red;
-    }
+      .validation-error {
+        font-size: 12px;
+        color: $validation-red;
+      }
 
-    textarea {
-      padding: 15px;
-      border-color: $border-color;
-      border-radius: 2px;
+      textarea {
+        padding: 15px;
+        border-color: $border-color;
+        border-radius: 2px;
+        overflow: hidden;
 
-      &.email-subject {
-        margin-bottom: 5px;
+        &.email-subject {
+          margin-bottom: 5px;
+        }
+      }
+
+      .email-automatic {
+        padding: 20px 0 0;
       }
     }
+  }
+}
+
+.email-type {
+  margin: 0 0 10px;
+  background-color: $email-composition-type;
+  padding: 15px;
+  border-radius: 8px;
 
-    .email-automatic {
-      padding: 20px 0;
+  .type-radio-group {
+    display: flex;
+    flex-direction: row;
```

Comment:
real quick change: I think we want to set `overflow: hidden` on this class, overwise the width of the radio options causes a scrollbar to appear.
![overflow-hidden](https://cloud.githubusercontent.com/assets/6207644/24720951/c80c6b06-1a0c-11e7-8348-b6be03106abb.png)